corehq/apps/locations/models.py

```diff
@@ -543,6 +544,7 @@ def location_type(self):
         return self.location_type_object.name
 
     _sql_location_type = None
+
     @location_type.setter
     def location_type(self, value):
```

Comment:
I *do* think though, that the majority of the instances where that rule is being applied are just mistakes where there *should* be a blank line.